src/test/java/stepDefsForWishList/StepsForWishList.java

```diff
@@ -0,0 +1,77 @@
+package stepDefsForWishList;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class StepsForWishList {
+    @Given("user is on landing page")
+    public void userIsOnLandingPage() {
+    }
+
+    @Given("user is logged in")
+    public void userIsLoggedIn() {
+    }
+
+    @And("user is on MY WISHLISTS page")
+    public void userIsOnMYWISHLISTSPage() {
+    }
+
+    @And("user type on name field My favourite")
```

Comment:
What if you want to type Abracadabra in name field? Try to do this step a little bit more flexible